build-aux/snap/snapcraft.yaml

```diff
@@ -546,12 +546,12 @@ parts:
         )
       elif [ -f "${CRAFT_STAGE}/fips-build" ]; then
         # grab the core22 from fips-updates channel
-        snap download --channel fips-updates/stable --basename core22-fips core22
+        snap download --channel fips-updates/edge --basename core24-fips core24
```

Comment:
Why do we have to switch to edge here?

Reply:
@Meulengracht @valentindavid what's holding back starting to push to other channels in fips-updates track? We're supposedly using the same libraries that are offered through a similar `pro` feature, and AFAIU those are considered 'stable' for general use